ontology/yaml/resources/HVAC/entity_types/VAV.yaml

```diff
@@ -611,6 +611,50 @@ VAV_DSP_PDSCV:
   - DSP
   - PDSCV
 
+VAV_CSP_DFVSC_DAIDC:
```

Comment:
this VAV doesn't have damper percentage control?

Reply:
It does not.